_posts/2021-09-27-changelog-156.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #156"
+date: 2021-09-27 13:00:00 +0300
+---
+
+This is the last release that supports the 2021.1 IDEs. To receive plugin updates further, please upgrade
+your IDE to 2021.2.
+
+## New Features
+
+* [#7317] Enable WSL support by default
+
+* [#7488] Add debugger option to break on panic. This option is enabled by default (by [@Kobzol])
+
+    TODO: GIF?
+
+* [#7550] Support finding usages in child modules in `Unused import` inspection
```

Comment:
Let's mention that it's still disabled by default not to produce false positives and describe a way on how to enable it